ledger/sandbox-classic/BUILD.bazel

```diff
@@ -380,9 +385,6 @@ server_conformance_test(
     servers = ONLY_POSTGRES_SERVER,
     test_tool_args = [
         "--open-world",
-        "--include=ParticipantPruningIT",
```

Comment:
These tests have been moved from wall-clock-time to static-time

Reply:
Right, I should have added them to `conformance-test-wall-clock-append-only` and I should also have removed this suite. Are they supposed to work only with wall clock (I suspect so)?

Reply:
085a15e